lib/test_bot.rb

```diff
@@ -21,6 +21,8 @@ module Homebrew
   module TestBot
     module_function
 
+    GIT = "/usr/bin/git".freeze
```

Comment:
```suggestion
    GIT = "/usr/bin/git"
```

already frozen by `# frozen_string_literal: true` above.